app/assets/stylesheets/components/buttons.scss

```diff
@@ -51,7 +51,15 @@ a.clear-bookmarks {
   margin: 10px 0;
 }
 
+.btn-outline-secondary {
+  color: $black;
+}
 
+.search-constraint-value:hover,
+.search-constraint-value > .btn-outline-secondary:active {
+  background-color: transparent;
+  color: $black;
+}
```

Comment:
maybe it's better to add:
  border-color: #ccc;
  &:hover {
    background-color: #ddd;
    color: $dark;
  }
this way the border color and the hover will be closer to the ones in production.

Reply:
^ this comment is for the `.btn-outline-secondary` line 54 I added in in the wrong line.